macros/sql/star.sql

```diff
@@ -23,7 +23,7 @@
 
     {%- for col in include_cols %}
 
-        {% if relation_alias %} {{ relation_alias }}.{% endif %}"{{ col }}" {% if not loop.last %},
+        {% if relation_alias %} {{ relation_alias }}.{% endif %} {{ col }} {% if not loop.last %},
```

Comment:
this change might break a lot of models! The `star` macro is frequently used on source tables, and those frequently have columns named things like `date` or `timestamp` that can be reserved words. Rather than always removing these quotes, can we add an argument called `quote` that defaults to True? That way, you can disable quoting where required.